source/reference/command/serverStatus.txt

```diff
@@ -3364,6 +3397,28 @@ shardingStatistics
 
    .. versionadded:: 5.0
 
+.. serverstatus:: shardingStatistics.numHostsTargeted
+
+   Indicates the number of shards targeted for ``CRUD`` operations and 
+   aggregation commands. When a ``CRUD`` operation or aggregation
+   command is run, the following metrics will be incremented.
+
+   .. include:: /includes/fact-server-status-num-hosts-targeted.rst
+
+   .. note:: Running the :dbcommand:`serverStatus` on :binary:`mongos` 
+            will provide insight into the aggregation operations find, 
+            insert, update, delete and aggregate that run on a sharded 
```

Comment:
Can we use a similar wording as done in the release notes and say something like:

```suggestion
            will provide insight into CRUD and aggregation operations that run on
            a sharded 
```

By saying "the aggregation operations find, insert, update, delete", it sounds like those are aggregation operators, which is a little misleading.

Reply:
Updated.